crates/primitives-traits/src/account.rs

```diff
@@ -169,6 +200,25 @@ impl From<AccountInfo> for Account {
             balance: revm_acc.balance,
             nonce: revm_acc.nonce,
             bytecode_hash: (code_hash != KECCAK_EMPTY).then_some(code_hash),
+            #[cfg(feature = "scroll")]
+            code_size: 0,
+            #[cfg(feature = "scroll")]
+            poseidon_code_hash: B256::ZERO,
```

Comment:
This is added as an intermediate solution in order for the `clippy` check to pass. Once we have migrated the whole code base to use our `ScrollAccountInfo`, we can remove this